package.json

```diff
@@ -39,6 +39,7 @@
     "semver": "^5.2.0",
     "touch": "^1.0.0",
     "update-json": "^1.0.0",
-    "yargs": "^4.7.1"
+    "yargs": "^4.7.1",
+    "replace-in-file": "^2.5.0"
```

Comment:
Thanks for the tip. I run the command and additionally it added a new line at the end of the file.